spec/controllers/token_based_resume_controller_spec.rb

```diff
@@ -31,14 +38,23 @@
   end
 
   describe "User tries to resume their application after email sent" do
+    # clean all the existing applications
+    # UnaccompaniedMinor.destroy_all
+
+    given_name = "First".freeze
+    email = "test@example.com".freeze
```

Comment:
Maybe refactor these to use the vars from the describe block above? I think I re-declared them here when experimenting 😊